public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -698,6 +699,10 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> impleme
     return this._serializer.getElementIdForPanel(panelId);
   }
 
+  public getDefaultDsReferencesForElement(elementId: string) {
```

Comment:
I can see how the `_serializer` makes sense to have it as public property. wdyt?